dev-server.py

```
@@ -0,0 +1,122 @@
+        print("아래 폴더가 보이지 않습니다 (프로젝트 루트에서 실행하세요):")
+        for m in missing:
+            print("  - " + m)
+        sys.exit(1)
+
+    socketserver.TCPServer.allow_reuse_address = True
+    with socketserver.TCPServer(("", port), MultiRootHandler) as httpd:
+        print("=" * 60)
+        print(" NUNCHI 프론트 미리보기 서버 시작")
+        print("=" * 60)
+        print(f" 포트       : {port}")
+        print(f" 정적 루트  :")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)